### DISCUSSION TITLE
Assigning the CALLS ALL button to a joystick key

### DISCUSSION THREAD

**Discussion by hanomax24** [flybywiresim/aircraft#6598] [Help]
Hi,
I want to assign the CALLS ALL Button to a joystick button. I found the Costum LVAR: PUSH_OVHD_CALLS_ALL. I use the registered version of FSUIPC, but I don't know how to bind the function to the button.

Any ideas?

Thx

**Comment by Mat7549** [comment]
Hi,

Personally I use '' joy to key '' because I don't know how to use fsuipc lol. It works very well!

Hope I answer your question,
See you in the sky,
Mat

**Comment by hanomax24** [comment]
That only works with commands where a key command exists. I want to press the Calls All-Button via joystick (or key).

Hano

**Reply by Mat7549** [reply]
Ah, I don't know if it's possible. Sorry!
